dracut/anaconda-lib.sh

```diff
@@ -452,3 +457,42 @@ wait_for_disks() {
     DISKS_WAIT_RETRIES=$((DISKS_WAIT_DELAY * 2))
     echo "[ \"\$main_loop\" -ge \"$DISKS_WAIT_RETRIES\" ]" > "$finished_hook"
 }
+
+# This script should start dnsconfd if all required conditions to run it are met
+start_dnsconfd() {
+
+    local reason="$1"
+    local start="yes"
+
+    echo "Attempting to start dnsconfd. Reason: ${reason}"
+
+    # dnsconfd is explicitly required by kernel boot option
+    dns_backend=$(getarg rd.net.dns-backend=)
+    if [ "${dns_backend}" != "dnsconfd" ]; then
+        echo "Attempting to start dnsconfd. Not starting because not required by kernel boot option."
+        start="no"
+    fi
+
+    # Network is required in initramfs
+    getargbool 0 rd.neednet && neednet=1
+    if [ ! -e "/tmp/net.ifaces" ] && [ "${neednet}" != "1" ]; then
+        echo "Attempting to start dnsconfd. Not starting because network is not required (yet)."
+        start="no"
+    fi
+
+    # It is not possible certificates for dnsconfd will be imported after start by kickstart
+    kickstart="$(getarg inst.ks=)"
+    # If kickstart has not been parsed yet && is reqiured by boot options
+    if [ ! -e /run/install/ks.cfg ] && ([ -n "$kickstart" ] || getargbool 0 inst.ks); then
+        echo "Attempting to start dnsconfd. Not starting because certificates can be imported via kickstart later."
+        start="no"
+    fi
+
+    if [ "${start}" == "yes" ]; then
+        echo "Attempting to start dnsconfd. Starting."
```

Comment:
I think you can just write it starting as this shouldn't be just an attempt.

Reply:
I wanted make easier matching the message to the previous related log message `Attempting to start dnsconfd. Reason: ${reason}` when looking at the log.